Added some scrolling shortcuts

iainhouston opened the pull request.

iainhouston committed:
Added some scrolling shortcuts

FichteFoll commented on source/reference/keyboard_shortcuts_osx.rst:
```diff
@@ -110,6 +110,21 @@ Find/Replace
++-----------------+-----------------------------------------------------------+
+| Keypress        | Command                                                   |
++=================+===========================================================+
+| ⌃ + V           | Scroll down one window                                    |
```
The terminology of a "window" is inappropriate here. Instead, use "view" or just "page". For ^L use just view.

Also, is there no binding for going upward?

FichteFoll commented on source/reference/keyboard_shortcuts_osx.rst:
```diff
@@ -110,6 +110,21 @@ Find/Replace
++=================+===========================================================+
+| ⌃ + V           | Scroll down one window                                    |
++-----------------+-----------------------------------------------------------+
+| ⌃ + L           | Centre current line vertically in window                  |
```
@guillermooo, do we have guidelines for american vs british english?

guillermooo replied:
I've used American spellings from the beginning, so let's stick to that.

FichteFoll replied:
Okay, please change this to `center` then, @iainhouston.

iainhouston committed:
Added some scrolling shortcuts

No, there's no "scroll up one page" that I have found. The ones I just added are those familiar old emacs key bindings which Sublime has been able to hang onto where they don't clash with OS shortcuts. So they're not a complete set ... but useful nevertheless. These and other emacs shortcuts are used all over the place including the OS X Terminal for editing the command line.

FichteFoll commented:
Thanks.
